components/src/core/Drawer/DrawerRailItem.tsx

```diff
@@ -32,35 +32,40 @@ export type DrawerRailItemClasses = {
 
 export type ExtendedNavItem = NavItem & { ButtonBaseProps?: Partial<MuiButtonBaseProps> };
 export type DrawerRailItemProps = SharedStyleProps & {
-    // classes for style overrides
+    /** Custom classes for default style overrides */
     classes?: DrawerRailItemClasses;
 
-    // toggles the condensed style
+    /** Sets whether to show the condensed variant without a title
+     *
+     * Default: false
+     */
     condensed?: boolean;
```

Comment:
I feel like the descriptions for this and the condensed prop on the drawer should be the same. And this makes me wonder why there is a prop for this here and one on the drawer instead of being in the shared props...